daprdocs/content/en/developing-applications/integrations/Azure/azure-kubernetes-service-extension.md

```diff
@@ -0,0 +1,89 @@
+
+{{% alert title="Note" color="warning" %}}
+If you install Dapr through the AKS extension, our recommendation is to continue using the extension for future management of Dapr instead of the Dapr CLI. Combining the two tools can cause conflicts and result in undesired behavior.
+{{% /alert %}}
+
+### How the extension works
+The Dapr extension uses the Azure CLI to provision the Dapr control plane on your AKS cluster. The Dapr control plane consists of:
+
+- **dapr-operator**: Manages component updates and Kubernetes services endpoints for Dapr (state stores, pub/subs, etc.)
+- **dapr-sidecar-injector**: Injects Dapr into annotated deployment pods and adds the environment variables DAPR_HTTP_PORT and DAPR_GRPC_PORT to enable user-defined applications to easily communicate with Dapr without hard-coding Dapr port values.
+- **dapr-placement**: Used for actors only. Creates mapping tables that map actor instances to pods
+- **dapr-sentry**: Manages mTLS between services and acts as a certificate authority. For more information read the security overview.
+
+### Extension Prerequisites 
+In order to use the AKS Dapr extension, you must first enable the `AKS-ExtensionManager` and `AKS-Dapr` feature flags on your Azure subscription.
+
+The below command will register the `AKS-ExtensionManager` and `AKS-Dapr` feature flags on your Azure subscription:
+
+```bash
+az feature register --namespace "Microsoft.ContainerService" --name "AKS-ExtensionManager"
+az feature register --namespace "Microsoft.ContainerService" --name "AKS-Dapr"
+```
+
+After a few minutes, check the status to show `Registered`. Confirm the registration status by using the az feature list command:
+
+```bash
+az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/AKS-ExtensionManager')].{Name:name,State:properties.state}"
+az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/AKS-Dapr')].{Name:name,State:properties.state}"
+```
+
+Next, refresh the registration of the `Microsoft.KubernetesConfiguration` and `Microsoft.ContainerService` resource providers by using the az provider register command:
+
+```bash
+az provider register --namespace Microsoft.KubernetesConfiguration
+az provider register --namespace Microsoft.ContainerService
+```
+
+#### Enable the Azure CLI extension for cluster extensions
+You will also need the `k8s-extension` Azure CLI extension. Install this by running the following commands:
+
+```bash
+az extension add --name k8s-extension
+```
+
+If the `k8s-extension` extension is already present, you can update it to the latest version using the below command:
+
+```bash
+az extension update --name k8s-extension
+```
+
+#### Create the extension and install Dapr on your AKS cluster
+Once your subscription is registered to use Kubernetes extensions, you can create the Dapr extension, which installs Dapr on your AKS cluster. For example:
```

Comment:
"After your subscription is registered to use Kubernetes extensions, install Dapr on your cluster by creating the Dapr extension. For example:"